x/validator/client/cli/query_proposed_disable_validator.go

```diff
@@ -0,0 +1,75 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/utils/cli"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/validator/types"
+)
+
+func CmdListProposedDisableValidator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "all-proposed-disable-validators",
+		Short: "Query the list of all proposed disable validators",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			params := &types.QueryAllProposedDisableValidatorRequest{
+				Pagination: pageReq,
+			}
+
+			res, err := queryClient.ProposedDisableValidatorAll(context.Background(), params)
```

Comment:
Check if a light client proxy is used for the list query (see other similar functions, e.g. `CmdListValidator`).